src/lib/util.coffee

```diff
@@ -13,45 +13,102 @@ extractOptsAndCallback = require('extract-opts')
 
 # =====================================
 # Export
+###*
+# The DocPad Util Class.
+# Collection of DocPad utility methods
+# @class docpadUtil
+# @constructor
```

Comment:
Is this the correct properties for a standard javascript object? As it is not a class or a constructor.